GraphQL/Services/ContentItemTypeBuilder.cs

```diff
@@ -0,0 +1,51 @@
+﻿using GraphQL.Types;
+using OrchardCore.ContentManagement.GraphQL.Queries.Types;
+using OrchardCore.ContentManagement.Metadata.Models;
+using System.Linq;
+
+namespace Lombiq.TrainingDemo.GraphQL.Services
+{
+    public class ContentItemTypeBuilder : IContentTypeBuilder
+    {
+        // It's a good practice to make the argument name a const because you will reuse it in the IGraphQLFilter.
+        public const string AgeFilterName = "age";
+
+        // Here you can add arguments to every Content Type (top level) field.
+        public void Build(
+            FieldType contentQuery,
+            ContentTypeDefinition contentTypeDefinition,
+            ContentItemType contentItemType)
+        {
+            // You can check the sub-fields if your argument relies on certain content parts or content fields.
+            if (contentItemType.Fields.All(field => field.Name != "person")) return;
+
+            // The resolved type can be anything that can be represented with JSON and has a known graph type, but we
+            // tick with numbers for simplicity's sake.

```

Comment:
?
```suggestion
            // stick with numbers for simplicity's sake.
```

Reply:
Ezeket amúgy egyből becommitolhatod innen is.

Reply:
Tudom, de ha már meg van nyitva a projekt akkor kézzel sokkal gyorsabb és utána nem kell git pullra se várni.